modules/cpmso-deleting.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * machine_management/cpmso-disabling.adoc
+
+:_content-type: PROCEDURE
+[id="cpmso-deleting_{context}"]
+= Deleting the control plane machine set
+
+To stop managing control plane machines with the control plane machine set on your cluster, you must delete the `ControlPlaneMachineSet` custom resource (CR).
+
+.Procedure
+
+* Delete the control plane machine set CR by running the following command:
++
+[source,terminal]
+----
+$ oc delete controlplanemachineset.machine.openshift.io cluster --namespace openshift-machine-api
+----
+
+.Verification
+
+* Check the control plane machine set custom resource status. A result of `Inactive` indicates that the removal and replacement process is successful. A `ControlPlaneMachineSet` CR exists but is not activated.
```

Comment:
Do we want to say state directly rather than status? Status typically referes to the status object within the CR which doesn't contain this information, the state is in the spec